standard/classes.md

```diff
@@ -4223,8 +4227,8 @@ The `true` and `false` unary operators require pair-wise declaration. A compile-
 > public class IntVector
 > {
 >     public IntVector(int length) {...}
->     public int Length { ... }           // Read-only property
->     public int this[int index] { ... }  // Read-write indexer
+>     public int Length { get { /* ... */ } }                          // Read-only property
+>     public int this[int index] { get { /* ... */ } set { /*...*/} }  // Read-write indexer
```

Comment:
Add space around the second ellipsis for consistency?

Reply:
Also remove all `/*` and `*/`, as they are added by the extractor.